golem-cli/src/clients/api_definition.rs

```diff
@@ -42,6 +48,70 @@ pub struct ApiDefinitionClientLive<C: golem_client::api::ApiDefinitionClient + S
     pub client: C,
 }
 
+#[derive(Debug, Copy, Clone)]
+enum Action {
+    Create,
+    Update,
+    Import,
+}
+
+impl Display for Action {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let str = match self {
+            Action::Create => "Creating",
+            Action::Update => "Updating",
+            Action::Import => "Creating or updating",
```

Comment:
That's a bit, I don't know. May be its ok, as there is no other way to detect

Reply:
It should be "Importing". I'll fix it. And it is just for logs.